Sources/Kumo/Errors/ResponseError.swift

```diff
@@ -1,5 +1,9 @@
 import Foundation
 
+#if canImport(KumoCoding)
```

Comment:
Is the `canImport` necessary? I think with this change Kumo depends entirely on KumoCoding so it should always be importable?

Reply:
Yes because cocoapods don’t allow multiple frameworks in a single podfiles, As far as I can tell.

Reply:
Ah I see, that makes sense.

I think in that case we would need to create a separate podspec for KumoCoding, similar to how RxSwift is setup with RxCocoa:

https://github.com/ReactiveX/RxSwift/blob/master/RxSwift.podspec
https://github.com/ReactiveX/RxSwift/blob/master/RxCocoa.podspec

That said, we can handle that in a future PR